src/main/java/com/tpwalke2/bluemapsignmarkers/core/bluemap/BlueMapAPIConnector.java

```diff
@@ -66,7 +67,8 @@ private void resetQueue() {
     }
 
     private void processMarkerAction(MarkerAction markerAction) {
-        LOGGER.info("Processing marker action: {}", markerAction);
+        if (!ConfigManager.get().isSilentLogs())
+            LOGGER.info("Processing marker action: {}", markerAction);
```

Comment:
If there are many signs in the configuration, I can see an argument for this causing too many entries on the server console. In the past (https://github.com/tpwalke2/BlueMapSignMarkers/pull/48), I have addressed this by simply changing to `LOGGER.debug`.

Using `LOGGER.debug` would then allow the log4j configuration to control whether these entries appeared in the server console.